### DISCUSSION TITLE
https://icylightdream.github.io/2022/03/11/%E6%89%8B%E5%B7%A5%E5%88%B6%E4%BD%9C%E4%BD%BF%E7%94%A8WinXShell%E7%9A%84PE-1/

### DISCUSSION THREAD

**Discussion by giscus** [IcyLightDream/IcyLightDream.github.io#9] [Announcements]
# https://icylightdream.github.io/2022/03/11/%E6%89%8B%E5%B7%A5%E5%88%B6%E4%BD%9C%E4%BD%BF%E7%94%A8WinXShell%E7%9A%84PE-1/

https://icylightdream.github.io/2022/03/11/%E6%89%8B%E5%B7%A5%E5%88%B6%E4%BD%9C%E4%BD%BF%E7%94%A8WinXShell%E7%9A%84PE-1/

**Comment by Chastinian** [comment]
PECMD下载地址：https://www.lanzoui.com/b279972

**Reply by IcyLightDream** [reply]
好家伙两年前写的抽象东西到现在还有人看
（~~不过这么个破博客站是怎么翻到的啊~~
